docs/ancestry.md

```diff
@@ -675,8 +656,48 @@ is {math}`L (1 - e^{-r})`
 and so {math}`Lr (1 - r/2) < L (1 - e^{-r}) < Lr`.
 :::
 
-For example, the following defines a recombination map for three chromosomes
-each 1 centiMorgan in length:
+(sec_ancestry_multiple_chromosomes)=
+
+### Multiple chromosomes
+
+Msprime does not directly support simulating multiple chromosomes
+simultaneously, but we can emulate it using a single linear genome split into
+chromosome segments. Multiple chromosomes are modelled by specifying
+a recombination map with single base-pair segments with recombination
+probability 1/2 separating adjacent chromosomes. The probability that
+a recombination breakpoint occurs at a given base pair that has recombination
+rate {math}`r` is {math}`1-e^{-r}`. Thus, we set the recombination rate in the
+base pair segments separating chromosomes to {math}`\log(2)`. This ensures that
+the recombination probability each generation is 1/2.
+
+#### Do I need to do this?
+
+The main purpose of simulating multiple chromosomes together (instead of
+running independent simulations for each chromosome) is to capture the correct
+correlation between trees on distinct chromosomes. In many scenarios,
+**simulating multiple chromosomes may not be necessary**, and independent
+simulations may be preferable as they will be much more efficient.
+
+[Nelson et al. 2020](https://doi.org/10.1371/journal.pgen.1008619) describes
+a few cases where simulating multiple chromosomes using the DTWF is important
+for accurately modeling shared patterns of variation between samples.
+Additionally, in large simulated cohorts, where many samples can be close
+relatives, we need the DTWF model and multiple chromosomes to obtain the
+correct distribution of the number and total length of shared IBD segments. We
+similarly require multiple chromosomes and the DTWF model to get the correct
+ancestry patterns for very recently admixed populations.
+
+Additionally,
+{ref}`Multiple merger coalescents <sec_ancestry_models_multiple_mergers>`
+result in positively correlated
+ancestries between unlinked chromosomes. This correlation does not break down
+in time and multiple chromosomes should not be simulated independently under
+the multiple merge model.
+
+#### Example
+
+The following defines a recombination map for three chromosomes each
+1 centiMorgan in length:
 
 ```{code-cell}
 import math
```

Comment:
Heh, whoops: this should actually be:
```
positions = [0, 1e6 - 1, 1e6, 2e6 - 1, 2e6, 3e6]
```

Reply:
Strangely, I think what is currently there puts the break point in the right place. The `[..., 1e6-1, 1e6, ...]` puts a break point at the positions `1e6 - 1`, so that you'd end up with the last base pair of the first chromosome (positions 999999) being unlinked from the chromosome it's supposed to be in and instead linked to the next chromosome. This can be seen by setting `r_chrom` in this example to zero, and checking number of trees per chromosome and their intervals.

It's a bit confusing to me, since I'd think that it would put the break point _between_ those positions, but the left inclusive interval setup and discrete genome means the break point is effective a base pair farther left than you'd expect.

Reply:
Ah, ok! Nice catch.